pkgs/by-name/gi/gittuf/package.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "gittuf";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "gittuf";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-lECvgagcqBS+BVD296e6WjjSCA3vI0nfLzpLTi/7N0I=";
+  };
+
+  vendorHash = "sha256-UKhXbZXKNtMnQe7sHBOmzzXGBHuDTYeZGKnteZirskA=";
+
+  doCheck = false;
```

Comment:
Could you please add a comment why check disabled.

Reply:
This works for me:
```nixx
  nativeCheckInputs = [ git ];
  checkFlags = [ "-skip=TestLoadRepository" ];
```
Not sure why TestLoadRepository failed.

Reply:
It looks to me like the test is supposed to be run in an initialized git repo (the gittuf repo).

I'd suggest to skip it for now and maybe clean it up upstream.

Reply:
How to validate this: 
```
$ git clone https://github.com/gittuf/gittuf.git
$ cd gittuf
$ go test github.com/gittuf/gittuf/internal/repository
ok  	github.com/gittuf/gittuf/internal/repository	0.618s
$ rm -r .git
$ go test github.com/gittuf/gittuf/internal/repository
[same error]
```

Reply:
I see, TestLoadRepository failed since it expects there is `.git` in the project root. So it should be fine to skip it.